docs/localization.rst

```diff
@@ -1,5 +1,3 @@
-.. _`Localization of Documentation`:
```

Comment:
This one is used in `faq.rst`

Reply:
Yeah, and was changed to use the `doc` directive as rtd recommends

https://github.com/rtfd/readthedocs.org/pull/3624/files#diff-c6f0a0722c6a2f86277535d7bcec7f8cR167